front/app/modules/commercial/project_folders/admin/containers/permissions/index.tsx

```diff
@@ -233,4 +233,4 @@ const FolderPermissions = ({
   );
 };
 
-export default injectIntl(FolderPermissions);
+export default withRouter(injectIntl(FolderPermissions));
```

Comment:
I'm not 100% sure about this... I'm not sure where the previous component was getting the withRouter components: 

```
    {
            path: 'permissions',
            name: 'admin projects edit folder permissions',
            container: () => import('./admin/containers/permissions'),
          },
```

And the previous component was with `injectIntl` without being wrapped in `withRouter`

Reply:
It comes from the route, see front/app/modules/commercial/project_folders/index.tsx. Lines 197 and 210.

Here too, you could use useParams.